dag/agriculture.yml

```
@@ -196,3 +196,33 @@ steps:
     - data://garden/demography/2024-07-15/population
   data://grapher/agriculture/2025-08-06/decoupling_food_production_and_land_use:
     - data://garden/agriculture/2025-08-06/decoupling_food_production_and_land_use
+  #
+  # Lobell Lab: Crop productivity in southern Africa
+  #
+  data://meadow/agriculture/2025-10-15/crop_yields_in_southern_africa:
+    - snapshot://agriculture/2025-10-15/crop_yields_in_southern_africa.zip
+  #
+  # Lobell Lab: Crop productivity in southern Africa
+  #
+  data://garden/agriculture/2025-10-15/crop_yields_in_southern_africa:
+    - data://meadow/agriculture/2025-10-15/crop_yields_in_southern_africa
+  #
+  # Lobell Lab: Crop productivity in southern Africa
+  #
+  data://grapher/agriculture/2025-10-15/crop_yields_in_southern_africa:
+    - data://garden/agriculture/2025-10-15/crop_yields_in_southern_africa
+  #
+  # USDA/FAS - Corn yields
+  #
+  data://meadow/agriculture/2025-10-16/corn_yields:
+    - snapshot://agriculture/2025-10-16/corn_yields.xls
+  #
+  # USDA/FAS - Corn yields
+  #
+  data://garden/agriculture/2025-10-16/corn_yields:
+    - data://meadow/agriculture/2025-10-16/corn_yields
+  #
+  # USDA/FAS - Corn yields
+  #
+  data://grapher/agriculture/2025-10-16/corn_yields:
+    - data://garden/agriculture/2025-10-16/corn_yields
```

let's group these together under the same comment





```suggestion
  #
  # USDA/FAS - Corn yields
  #
  data://meadow/agriculture/2025-10-16/corn_yields:
    - snapshot://agriculture/2025-10-16/corn_yields.xls
  data://garden/agriculture/2025-10-16/corn_yields:
    - data://meadow/agriculture/2025-10-16/corn_yields
  data://grapher/agriculture/2025-10-16/corn_yields:
    - data://garden/agriculture/2025-10-16/corn_yields
```

---

The limitation is just about how we write to the DAG (so, nothing important about processing). We have functions that can read the comment right above a step, and write a comment right above a step. But figuring out which comment to fetch when steps are grouped, and how to group similar steps, are difficult tasks.